xml/System.Diagnostics/ActivityContext.xml

```diff
@@ -41,13 +41,21 @@
         <Parameter Name="isRemote" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="traceId">To be added.</param>
-        <param name="spanId">To be added.</param>
+        <param name="traceId">A trace identifier.</param>
+        <param name="spanId">A span identifier.</param>
         <param name="traceOptions">To be added.</param>
```

Comment:
><param name="traceOptions">To be added.</param> [](start = 8, length = 47)

An 8-bit field that controls tracing flags such as sampling, trace level, etc.

Reply:
```suggestion
        <param name="traceOptions">An 8-bit field that controls tracing flags such as sampling, trace level, etc.</param>
```